openstudio_measure_tester.gemspec

```diff
@@ -37,6 +37,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rubocop-checkstyle_formatter', '~> 0.4.0'
   spec.add_dependency 'simplecov', '~> 0.18.2'
```

Comment:
latest avail: rubocop 1.9.1, simplecov 0.21.2

Reply:
newer versions of rubocop have a native dep with jaro-winkler. Despite openstudio cli having jaro_winkler, it still caused problems and we had to roll back tor ubocop (0.54.0).

Reply:
yep, I think we are still stuck on 0.54.0